draft-ietf-asdf-nipc.mkd

```diff
@@ -96,50 +96,48 @@ a particular protocol, such as a BLE GATT REST API for BLE Gateways
 a particular use case. In absence of an open standard describing how
 applications on an IP network communicate with non-IP devices, bespoke and
-vendor-specific implementations have proliferated. This results in parallel
-infrastructure of both gateways and non-IP networks being deployed on a case
-by case basis, each connecting separately into the IP network, with a distinct
-set of APIs. At the same time, wireless access points supporting both IP-based 
-wireless connectivity as well as non-IP based wireless technologies are
-deployed ubiquitiously. Many of these wireless access points are equipped with
-radios that can transmit and receive different frame types, such as {{BLE53}}
-and {{Zigbee22}}. This specification aims to define a Gateway API for these 
-Non-IP protocols that can be leveraged by this wireless infrastructure in order
-to connect Non-IP devices into IP networks. The specification aims to be
+vendor-specific implementations have proliferated. The implication of this is
+that every use case deployment requires implementation of a new API, and possibly
+requires its own gateway, which results in proliferation of network infrastructure.
+This specification aims to define a Gateway API for these Non-IP protocols
+that can be leveraged by a Network Gateway connected to a wireless network. 
+The API aims to enable a Gateway to support any non-IP protocol. The gateway
+can simultaneously support multiple applications and multiple non-IP protocols
+as well as different versions of the same protocol. The specification is
 extensible, in order to support existing and future non-IP protocols.
 
 A standardized Non-IP Gateway interface has following benefits:
 
- - Avoid the need for parallel Non-IP infrastructure.
- - Avoid the need for applications to perform bespoke integrations for
+ - Avoid the need for applications to perform new and bespoke integrations for
    different environments.
+ - Avoid having to deploy multiple Gateway functions to support multiple non-IP use cases.
  - Faster and more cost-effective adoption of Non-IP devices in IP network
    environments.
 
 ## Non-IP Gateway
 
-A non-IP gateway MUST provide at least following functions:
+A non-IP gateway is an application layer gateway that allows Applications on IP networks to communicate with Non-IP devices leveraging NIPC APIs.
+NIPC APIs consist of reading or writing properties of devices, performing actions on devices, as well as enabling or disabling events on devices.
+
+In order to perform NIPC operations on a device, 2 prerequisites must be fulfilled:
+- The device must be onboarded on the gateway. Onboarding declares a device instance to the gateway. This allows the NIPC Gateway to retrieve the device object, identified by an id referenced in the path of the NIPC API. The definition of the onboarding function is out of scope of this document, but can be provided by a provisioning interface such as {{!RFC7644}} leveraging {{!I-D.ietf-scim-device-model}}.
```

Comment:
> This allows the NIPC Gateway to retrieve the device object

"retrieve" and "the device object" are a bit confusing terms here, especially since "the device object" has not been introduced. Isn't the key point that when the Gateway receives an ID from the application it can associate it with information provided during the onboarding?